popde/density_estimate.py

```diff
@@ -230,7 +230,8 @@ def calc_ndim_bandwidth(self, method='oned_isj'):
         if method == 'oned_isj':
             from KDEpy.bw_selection import improved_sheather_jones as isj
             for i, col in enumerate(self.kde_data.T):
-                nd_bws[i] = isj(col)  # Weighting is also possible, not implemented atm
+                #col should be a data of shape ndata, ndim==1
```

Comment:
This comment is hard to understand, 'col' is what it is but we need to give the KDEpy function a specific shape of input
```suggestion
                # KDEpy function requires an array of shape (n_samples, 1)
```